CMakeLists.txt

```diff
@@ -25,13 +25,15 @@ endif()
 
 option( SPAN_LITE_OPT_BUILD_TESTS    "Build and perform span-lite tests" ${span_IS_TOPLEVEL_PROJECT} )
 option( SPAN_LITE_OPT_BUILD_EXAMPLES "Build span-lite examples" OFF )
+option( SPAN_LITE_EXPORT_PACKAGE     "Export span-lite package globally" ${span_IS_TOPLEVEL_PROJECT} )
```

Comment:
I wasn't exactly sure what to put for the description. `Export xx package globally` or `Export xx in the package registry` or something similar is what I usually put there.